src/MudBlazor/Components/Autocomplete/MudAutocomplete.razor.cs

```diff
@@ -976,7 +977,11 @@ private async Task OnInputActivationAsync(bool openMenu)
 
             if (openMenu)
             {
-                await OpenMenuAsync();
+                await DisposeTimerAsync();
+                if (DebounceInterval <= 0)
+                    _debounceTimer = new Timer(OnDebounceComplete, null, 5, Timeout.Infinite);
+                else
+                    _debounceTimer = new Timer(OnDebounceComplete, null, DebounceInterval, Timeout.Infinite);
```

Comment:
```suggestion
                var debounce = Math.Max(5, DebounceInterval); // The click and focus events will conflict with each other if the debounce is too low.
                _debounceTimer = new Timer(OnDebounceComplete, null, debounce, Timeout.Infinite);
```

adds comment + forces minimum of 5